tigerpath/majors_and_certificates/majors/ANT.json

```diff
@@ -10,19 +10,9 @@
   ],
   "contacts": [
     {
-      "type": "Chair",
+      "type": "Chair, Director of Undergraduate Studies",

```

Comment:
https://ua.princeton.edu/academic-units/department-department-anthropology

"Chair
Carolyn M. Rouse
Director of Undergraduate Studies
Carolyn M. Rouse
"

Reply:
I think we should keep the "Chair" and "Director of Undergraduate Studies" fields separate. Right now they're the same person, but it's possible it could be different people again in the future.